landlab/bmi/bmi_bridge.py

```diff
@@ -243,17 +253,40 @@ def __init__(self):
             self._clock = None
             super(BmiWrapper, self).__init__()
 
+            self._input_var_names = tuple(
+                set(self._cls._input_var_names) | {"boundary_condition_flag"}
+            )
+            # self._cls._input_var_names = tuple(
+            #     set(self._cls._input_var_names) | {"boundary_condition_flag"}
+            # )
+            self._output_var_names = tuple(
+                set(self._cls._output_var_names) | {"boundary_condition_flag"}
+            )
+            # self._cls._output_var_names = tuple(
+            #     set(self._cls._output_var_names) | {"boundary_condition_flag"}
+            # )
+            self._var_mapping = self._cls._var_mapping.copy()
+            self._var_units = self._cls._var_units.copy()
+            self._var_doc = self._cls._var_doc.copy()
+
+            self._var_mapping["boundary_condition_flag"] = "node"
+            self._var_units["boundary_condition_flag"] = ""
```

Comment:
is "-" more appropriate? I think we've used that in the past. Or is "" the correct _no units_ value for UDUNITS?

Reply:
"-" is an invalid unit for *cfunits*, so I've been trying to use the empty string to indicate "no units".